packages/ui/src/views/repo/repo-settings/components/repo-settings-general-rules.tsx

```diff
@@ -25,8 +25,10 @@ const Description = ({
 }) => {
   return (
     <Text color="tertiaryBackground" as="div" className="flex items-center gap-1 pl-[24px]">
-      {targetPatternsCount} target patterns <span className="text-2xl text-tertiary">|</span> {rulesAppliedCount} rules
-      applied <span className="text-2xl text-tertiary">|</span>
+      {targetPatternsCount} target patterns
+      <span className="pointer-events-none h-3 w-px bg-borders-2 ml-1 mr-1" aria-hidden />
+      {rulesAppliedCount} rules applied
```

Comment:
Shud we put this in en.json

Reply:
Lets take up i18n in `repo-settings` and `branch-rules` together as a separate PR?